keyboards/sck/m0116b/keymaps/m0118/keymap.c

```diff
@@ -0,0 +1,118 @@
+		KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,   KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_ENT,  KC_P7,   KC_P8,   KC_P9,   KC_PPLS, 
+		KC_LCAP, KC_A,    KC_S,    KC_D,    KC_F,   KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_HASH,          KC_P4,   KC_P5,   KC_P6,	  KC_PMNS,
+		KC_LSFT, KC_LABK, KC_Z,    KC_X,    KC_C,   KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_P1,   KC_P2,   KC_P3,   KC_PENT, 
+		KC_LCTL, KC_LGUI, KC_LALT,                           KC_SPC,                             KC_RALT, KC_LEFT, KC_RGHT, KC_DOWN, KC_P0,            KC_PDOT
+),
+/* Keymap Layer 1: (Layer 1) Layer 1
+   * ,---------------------------------------------------------------------------------.
+   * |                      |  Reset  |                                                |
+   * `---------------------------------------------------------------------------------'
+   * ,-----------------------------------------------------------. .-------------------.
+   * |   |   |   |   |   |   |   |   |   |   |   |   |   |       | |    |    |    |    |
+   * |-----------------------------------------------------------| |-------------------|
+   * |     |   |   |   |   |   |   |   |   |   |   |   |   |     | |    |    |    |    |
+   * |--------------------------------------------------------|  | |-------------------|
+   * |       |   |   |   |   |   |   |   |   |   |   |   |    |  | |    |    |    |    |
+   * |-----------------------------------------------------------' |-------------------|
+   * |     |  |   |   |   |   |   |   |   |   |   |   |     |    | |    |    |    |    |
+   * |-----------------------------------------------------------' `--------------|    |
+   * |     |     |     |     SAFE_RANGE     |    |    |     |    | |         |    |    |
+   * `-----------------------------------------------------------' `-------------------'
+   */
+[1] = LAYOUT_m0118_iso( /* Layer 1 */
+	                                                        RESET, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS,                            SAFE_RANGE,                         KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS
```

Comment:
```suggestion
		KC_TRNS, KC_TRNS, KC_TRNS,                            M0116B,                             KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS
```

Reply:
None of the keymaps  compile when using `M0116B` instead of `SAFE_RANGE`. they all compile perfectly fine, and the sendstring works correctly, when using `SAFE_RANGE`, which is why that is what is in in those spaces

Reply:
corrected